package.json

```diff
@@ -45,6 +45,10 @@
     "@angular/compiler-cli": "~10.0.0",
     "@angular/language-service": "~10.0.0",
     "@capacitor/cli": "2.4.2",
+    "@dvsa/mes-config-schema": "^1.1.0",
+    "@dvsa/mes-journal-schema": "^1.2.0",
+    "@dvsa/mes-search-schema": "^1.1.2",
+    "@dvsa/mes-test-schema": "^3.31.0",
```

Comment:
Get the package versions without the caret symbol (^). You can achieve that by adding the `--save-exact` flag to the `npm i`

i.e: `npm install @dvsa/mes-test-schema@latest --save-dev --save-exact`